controllers/warehousesController.js

```diff
@@ -0,0 +1,108 @@
+const Joi = require("joi");
+const knex = require("knex")(require("../knexfile"));
+
+// {
+//     "warehouse_id": 1,
+//     "item_name": "Paper Towels",
+//     "description": "Made out of military-grade synthetic materials, these paper towels are highly flammable, yet water resistant, and easy to clean.",
+//     "category": "Gear",
+//     "status": "Out of Stock",
+//     "quantity": 0
+//   }
+
+// POST -- Create New InveNtory of Warehouse by its ID
+const createNewWarehouse = async (req, res) => {
+  try {
+    // inventorySchema for req body validations
+    const warehouseSchema = Joi.object({
+      warehouse_name: Joi.string().required(),
+      address: Joi.string().required(),
+      contact_name: Joi.string().required(),
+      contact_information: Joi.string().required(),
+      actions: Joi.string().required().required(),
+    });
+
+    const { error } = warehouseSchema.validate(
+      //convert status from REQ BODY to lowerCase to pass Validation
+      { ...req.body, status: req.body.status.toLowerCase() },
```

Comment:
status is not part of the req.body values there for you don't have to add this line
` (status: req.body.status.toLowerCase()) `
status is a value for inventory

Reply:
updated thanks!